include/nbl/video/IGPUCommandBuffer.h

```diff
@@ -523,6 +524,20 @@ class NBL_API2 IGPUCommandBuffer : public IBackendObject
         };
         bool resolveImage(const IGPUImage* const srcImage, const IGPUImage::LAYOUT srcImageLayout, IGPUImage* const dstImage, const IGPUImage::LAYOUT dstImageLayout, const uint32_t regionCount, const SImageResolve* const pRegions);
 
+        bool IGPUCommandBuffer::setRayTracingPipelineStackSize(uint32_t pipelineStackSize);
+        bool IGPUCommandBuffer::traceRays(
+          const asset::SBufferRange<const IGPUBuffer>& raygenGroupRange, uint32_t raygenGroupStride,
+          const asset::SBufferRange<const IGPUBuffer>& missGroupsRange, uint32_t missGroupStride,
+          const asset::SBufferRange<const IGPUBuffer>& hitGroupsRange, uint32_t hitGroupStride,
+          const asset::SBufferRange<const IGPUBuffer>& callableGroupsRange, uint32_t callableGroupStride,
+          uint32_t width, uint32_t height, uint32_t depth);
+        bool IGPUCommandBuffer::traceRaysIndirect(
```

Comment:
add a `tradeRaysIndirect` overload which only takes an indirect buffer binding, in the style of `VK_KHR_ray_tracing_maintenance1`

you'll need to check device limits if `VK_KHR_ray_tracing_maintenance1` is available (it might be called something else, please merge `master` first there's some updates there)